server/src/routes/game.ts

```diff
@@ -46,30 +46,30 @@ export const gameRoutes = {
         })),
     )
     .mutation(async ({ input: { gameId, oldGameId } }) => {
-      addGame(gameId)
+      await addGame(gameId)
```

Comment:
We should add https://eslint.org/docs/latest/rules/require-await to make sure we didn't miss anything